__tests__/releaseNotesFormatter.test.js

```diff
@@ -0,0 +1,373 @@
+import test from 'ava'
+import addPRToRelease from '../src/helpers/releaseNotesFormatter'
+
+const BUG_RELEASE = `
+### Bug fixes
+
+#7 fix what was broken`
+
+const FEATURES_RELEASE = `
+### Features
+
+#5 add an awesome feature!!`
+
+const FEATURES_RELEASE_BREAKING = `${FEATURES_RELEASE}
+
+### BREAKING CHANGE
+
+* /ping route is removed (from #5)`
+
+const CHORES_RELEASE = `
+### Other changes
+
+#111 something boring!!`
+
+test('adds feature to empty release', async t => {
+  const release = await addPRToRelease({
+    title: 'feat: add the next big thing (#1000)',
+    description: '',
+    number: 1000
+  }, '')
+  t.is(release, `
+### Features
+
+#1000 add the next big thing`)
+})
+
+test('adds another feature to an existing feature list', async t => {
+  const release = await addPRToRelease({
+    title: 'feat: add the next big thing (#1000)',
+    description: '',
+    number: 1000
+  }, FEATURES_RELEASE)
+  t.is(release, `
+### Features
+
+#1000 add the next big thing
```

Comment:
I'd like to have the PR number at the end. I'd like to think of it as "What was changed (how it changed)". Implementation details are less important. Wouldn't it also be nice to have it in list form? My thinking is `- Add the next big thing (#1000)`

Reply:
👍